tools/profileBuilder/Gopkg.toml

```diff
@@ -1,50 +1,7 @@
-
-# Gopkg.toml example
-#
-# Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
-# for detailed Gopkg.toml documentation.
-#
-# required = ["github.com/user/thing/cmd/thing"]
-# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
-#
-# [[constraint]]
-#   name = "github.com/user/project"
-#   version = "1.0.0"
-#
-# [[constraint]]
-#   name = "github.com/user/project2"
-#   branch = "dev"
-#   source = "github.com/myfork/project2"
-#
-# [[override]]
-#  name = "github.com/x/y"
-#  version = "2.4.0"
-
-
-[[constraint]]
-  name = "github.com/marstr/collection"
-  version = "1.0.1"
-
-[[constraint]]
-  branch = "master"
-  name = "github.com/marstr/goalias"
-
-[[constraint]]
-  branch = "master"
-  name = "github.com/marstr/randname"
-
-[[constraint]]
-  branch = "master"
-  name = "github.com/mitchellh/go-homedir"
-
 [[constraint]]
   name = "github.com/spf13/cobra"
   version = "0.0.1"
```

Comment:
>"0.0.1 [](start = 12, length = 6)

there is 0.0.3 if we care to update. kind of sucks the guy doesnt provide any release notes.

Reply:
Default behavior is this version is treated as `^`, so it's the minimum required version.  So when you do `dep ensure` for the first time or `dep ensure -update` the tool will find the latest version that respects `^`.  If you look in the lock file you'll see that we're using 0.0.3.